cfme/rest/gen_data.py

```diff
@@ -431,20 +440,22 @@ def tenants(request, appliance, num=1):
     return tenants
 
 
-def users(request, appliance, num=1):
+def users(request, appliance, num=1, **kwargs):
     data = []
     for _ in range(num):
         uniq = fauxfactory.gen_alphanumeric(4).lower()
-        data.append({
-            "userid": "user_{}".format(uniq),
-            "name": "name_{}".format(uniq),
-            "password": fauxfactory.gen_alphanumeric(),
-            "email": "user@example.com",
-            "group": {"description": "EvmGroup-user_self_service"}
-        })
+        data.append(
+            {
+                "userid": kwargs.get("userid", "user_{}".format(uniq)),
+                "name": kwargs.get("name", "name_{}".format(uniq)),
+                "password": kwargs.get("password", fauxfactory.gen_alphanumeric()),
+                "email": kwargs.get("email", "{}@example.com".format(uniq)),
+                "group": {"description": kwargs.get("group", "EvmGroup-user_self_service")},
```

Comment:
@valaparthvi 
Do you need ```example.com``` in your email here?
If not we can directly use:
```
In [4]: mail = fauxfactory.gen_email()

In [5]: mail
Out[5]: 'swqFTFFK@test.gov'


```

Reply:
gen_email() will generate a completely different email, I think it's better if the email has some indication of which user it belongs to.